ext/opentelemetry-ext-dbapi/src/opentelemetry/ext/dbapi/__init__.py

```diff
@@ -208,14 +219,23 @@ def __init__(
                 "host": "host",
                 "user": "user",
             }
-        self.tracer = tracer
+        self._name = name
+        self._version = version
+        self._tracer_provider = tracer_provider
         self.database_component = database_component
         self.database_type = database_type
         self.connection_props = {}
         self.span_attributes = {}
         self.name = ""
         self.database = ""
 
+    def get_tracer(self):
```

Comment:
can this be memoized? should be object specific?

Reply:
Same here.